File: web3rpc/rpc-specs/code-samples/python/net/peerCountByType.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

w3 = Web3(Web3.HTTPProvider(host))
net_response = w3.net.peer_count_by_type()

print(net_response)

File: web3rpc/rpc-specs/code-samples/javascript/net/peerCountByType.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");

    provider.net.peerCountByType({}, (err, data, response) => {
        console.log(data);
    });
}
)()